editor/editor_help.cpp

```diff
@@ -761,7 +761,9 @@ void EditorHelp::_update_doc() {
 
 	_init_colors();
 
-	DocData::ClassDoc cd = doc->class_list[edited_class]; //make a copy, so we can sort without worrying
+	DocData::ClassDoc cd;
+	if (doc->class_list.has(edited_class)) //check to prevent empty string slipping into class_list
```

Comment:
AFAIK the whole function is moot if this condition is false, so it would be better to just have
```cpp
if (doc->class_list.has(edited_class)) {
    return;
```
at the very top of the method.

Reply:
You mean 

`if (!doc->class_list.has(edited_class)) {
    return;`

With ! right?

Reply:
Ah sure :D

Reply:
Added that line.